lib/util/auth.js

```diff
@@ -6,8 +6,15 @@ import { ACCOUNT_PATH, PERSISTENCE_STRATEGY_OTP_MIDDLEWARE } from './constants'
  */
 export const accountLinks = [
   {
-    text: 'My account',
+    messageId: 'components.NavLoginButton.myAccount',
```

Comment:
Do not create a messageId outside the context of a call to `intl.format` or a `FormattedMessage` component as it will not be readable by cli tools that find all message IDs.